src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -1542,6 +1542,15 @@ module Mutations = struct
     let%bind fee =
       result_of_exn Currency.Fee.of_uint64 fee ~error:"Invalid `fee` provided."
     in
+    let%bind () =
+      Result.ok_if_true
+        Currency.Fee.(fee >= User_command.minimum_fee)
+        ~error:
+          (sprintf
+             !"Invalid user command. Fee %d is less than the minimum fee, %d."
+             (Currency.Fee.to_int fee)
+             (Currency.Fee.to_int User_command.minimum_fee))
```

Comment:
It's possible we mess this up in other places, but it's probably safer to use `Currency.Fee.to_string` here instead of converting to an int if possible (in case the fee is ever bigger than int).

Reply:
Done